src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -1944,80 +1874,54 @@ namespace HVACMultiSpeedHeatPump {
         // determine the bypass fraction. The simulation converges quickly on mass flow rate. If the zone
         // temperatures float in the deadband, additional iterations are required to converge on mass flow rate.
 
-        // Using/Aliasing
-        using DataSizing::AutoSize;
-        using Fans::GetFanIndex;
-        using Fans::GetFanVolFlow;
-        using FluidProperties::GetDensityGlycol;
-        using FluidProperties::GetSatDensityRefrig;
-
-        using DXCoils::GetDXCoilAvailSchPtr;
-        using PlantUtilities::InitComponentNodes;
-        using PlantUtilities::ScanPlantLoopsForObject;
-        using PlantUtilities::SetComponentFlowRate;
-        using Psychrometrics::PsyRhoAirFnPbTdbW;
-        using ScheduleManager::GetCurrentScheduleValue;
-        using SteamCoils::SimulateSteamCoilComponents;
-        using WaterCoils::GetCoilMaxWaterFlowRate;
-        using WaterCoils::SimulateWaterCoilComponents;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         static constexpr std::string_view RoutineName("InitMSHeatPump");
-        int InNode;     // Inlet node number in MSHP loop
-        int OutNode;    // Outlet node number in MSHP loop
-        int ZoneInNode; // Zone inlet node number in the controlled zone for MSHP
-        Real64 RhoAir;  // Air density at InNode
+        Real64 RhoAir; // Air density at InNode
 
         Real64 QSensUnitOut; // Output of MSHP system with coils off
         Real64 PartLoadFrac; // Part-load ratio
         int ZoneNum;
-        int i;                 // Index to speed
-        int NumOfSpeedCooling; // Number of speeds for cooling
-        int NumOfSpeedHeating; // Number of speeds for heating
-        Real64 DeltaMassRate;  // Difference of mass flow rate between inlet node and system outlet node
+        int i;                // Index to speed
+        Real64 DeltaMassRate; // Difference of mass flow rate between inlet node and system outlet node
 
-        int ZoneInSysIndex(0);                            // number of zone inlet nodes counter in an airloop
         int NumAirLoopZones(0);                           // number of zone inlet nodes in an air loop
-        int ZoneInletNodeNum(0);                          // zone inlet nodes node number
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:1979]:(style),[unreadVariable],Variable 'ZoneInSysIndex' is assigned a value that is never used.
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:1981]:(style),[unreadVariable],Variable 'ZoneInletNodeNum' is assigned a value that is never used.